language_service/src/lib.rs

```diff
@@ -7,7 +7,6 @@
 mod compilation;
 pub mod completion;
 pub mod definition;
-mod display;
 pub mod hover;
```

Comment:
Does anything use this module/file now in this crate? I don't see the file being deleted.

Reply:
Not sure if this is an outdated comment. The display module is removed from the language server.